core/components/atoms/alert/alert.js

```diff
@@ -10,6 +10,15 @@ import { deprecate } from '../../_helpers/custom-validations'
 import Automation from '../../_helpers/automation-attribute'
 import Icon, { __ICONNAMES__ } from '../icon'
 
+const createIconForAlert = (icon, color) => {
```

Comment:
Why not do this in the "icon" component itself? It might break some components but will make the "icon" component universal and we will not have to do this in future when adding more components?

Reply:
I think this could be a helper that components requiring icons as a prop can consume.

Reply:
Yeah, I think that's better. Won't break anything, and won't be required to do again.

Reply:
@francocorreasosa just opened #979